src/javase02/task02/Worker.java

```diff
@@ -0,0 +1,57 @@
+package javase02.task02;
+
+import java.util.Scanner;
+
+public class Worker {
+    String name;
+    int numberOfChancellery;
```

Comment:
seems like numberOfChancellery no needed. chancelleries.length is enough

Reply:
yup